### DISCUSSION TITLE
How to annotate an array form-urlencoded parameter?

### DISCUSSION THREAD

**Discussion by avalloneandrea** [smallrye/smallrye-open-api#993] [Q&A]
Hi! I'm having trouble with this REST endpoint.

    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
    @Produces(MediaType.APPLICATION_JSON)
    public Response addItems(@Parameter(description = "Items to add", style = ParameterStyle.FORM, explode = Explode.TRUE) @FormParam("items") List<Long> items) {
        return service.addItems(items);
    }

this generates the following spec:

    schema:
      type: object
      properties:
        items:
          type: array
          items:
            format: int64
            type: integer

which is missing the encoding part

    encoding:
      items:
        style: form
        explode: true

and therefore create trouble with Swagger. It seems to me that every style/explode combination is just ignored. Is it the expected behavior or a bug?

**Comment by phillip-kruger** [comment]
@MikeEdgar ^^^ ?

**Reply by MikeEdgar** [reply]
@avalloneandrea , @phillip-kruger - I'll review this today. At first glance, it looks like something that is not accounted for.

**Reply by phillip-kruger** [reply]
Thanks @MikeEdgar !!

**Reply by codespearhead** [reply]
@avalloneandrea Does this still occur with v3.11.0?
